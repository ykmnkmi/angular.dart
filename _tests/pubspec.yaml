name: _tests
description: Tests for AngularDart.
publish_to: none

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  build: ^2.4.1
  build_resolvers: ^2.4.2
  build_test: ^2.2.2
  collection: ^1.19.1
  glob: ^2.1.2
  js: ^0.7.1
  logging: ^1.3.0
  ngcompiler: ^3.0.0-dev.3
  ngdart: ^8.0.0-dev.4
  ngtest: ^5.0.0-dev.2
  source_gen: ^1.5.0
  test: ^1.25.9

dev_dependencies:
  analyzer: ^6.5.0
  build_runner: ^2.4.12
  build_web_compilers: ^4.0.11
  csslib: ^1.0.2
  lints: ^5.0.0
  meta: ^1.16.0
  mockito: ^5.4.4
  package_config: ^2.1.1
  source_span: ^1.10.1
  term_glyph: ^1.2.1
