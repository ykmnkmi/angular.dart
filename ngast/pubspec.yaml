name: ngast
description: Parser and utilities for AngularDart templates
version: 3.0.0-dev.2
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  charcode: ^1.4.0
  collection: ^1.19.1
  meta: ^1.16.0
  source_span: ^1.10.1
  string_scanner: ^1.4.0

dev_dependencies:
  lints: ^5.0.0
  path: ^1.9.1
  test: ^1.25.9
