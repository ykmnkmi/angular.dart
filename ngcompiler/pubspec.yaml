name: ngcompiler
description: >
  The AngularDart compiler.
  It compiles your component files written in Dart into templates so that you
  don't have to worry about all the complicated hassle.
version: 3.0.0-dev.3
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  analyzer: ^6.5.0
  args: ^2.6.0
  build: ^2.4.1
  code_builder: ^4.10.1
  collection: ^1.19.1
  csslib: ^1.0.2
  dart_style: ^2.3.7
  logging: ^1.3.0
  meta: ^1.16.0
  ngast: ^3.0.0-dev.2
  ngdart: ^8.0.0-dev.4
  path: ^1.9.1
  source_gen: ^1.5.0
  source_span: ^1.10.1

dev_dependencies:
  build_test: ^2.2.2
  lints: ^5.0.0
  package_config: ^2.1.1
  test: ^1.25.9
