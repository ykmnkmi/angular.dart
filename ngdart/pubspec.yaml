name: ngdart
description: >-
  Fast and productive web framework by Google.
  Separate, but similar to the JavaScript framework with a similar name.
version: 8.0.0-dev.4
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  build: ^2.4.1
  built_collection: ^5.1.1
  built_value: ^8.9.2
  collection: ^1.19.1
  intl: ^0.19.0
  js: ^0.7.1
  meta: ^1.16.0
  ngcompiler: ^3.0.0-dev.3
  stream_transform: ^2.1.0

dev_dependencies:
  lints: ^5.0.0

  # minimal versions to support null safety and class modifiers
  build_runner: ^2.4.12
  build_web_compilers: ^4.0.11
  watcher: ^1.1.0
