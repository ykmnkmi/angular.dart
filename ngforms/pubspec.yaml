name: ngforms
description: >
  Forms framework for AngularDart.
  Easily create forms to accept input from users.
version: 5.0.0-dev.3
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  meta: ^1.16.0
  ngdart: ^8.0.0-dev.4

dev_dependencies:
  build_runner: ^2.4.12
  build_test: ^2.2.2
  build_web_compilers: ^4.0.11
  lints: ^5.0.0
  mockito: ^5.4.4
  ngtest: ^5.0.0-dev.2
  test: ^1.25.9
