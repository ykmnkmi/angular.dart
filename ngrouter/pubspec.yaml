name: ngrouter
description: >
  Router for AngularDart.
  It enables navigation from one view to the next as users perform tasks.
version: 4.0.0-dev.3
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  collection: ^1.19.1
  ngdart: ^8.0.0-dev.4

dev_dependencies:
  async: ^2.12.0
  build_runner: ^2.4.12
  build_test: ^2.2.2
  build_web_compilers: ^4.0.11
  lints: ^5.0.0
  mockito: ^5.4.4
  ngtest: ^5.0.0-dev.2
  test: ^1.25.9
