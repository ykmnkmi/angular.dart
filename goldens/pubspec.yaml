name: goldens
description: Goldens for AngularDart.
publish_to: none

environment:
  sdk: ">=3.6.0 <4.0.0"

resolution: workspace

dependencies:
  js: ^0.7.1
  meta: ^1.16.0
  ngdart: ^8.0.0-dev.4

dev_dependencies:
  build_runner: ^2.4.12
  build_web_compilers: ^4.0.11
  dart_style: ^2.3.7
  glob: ^2.1.2
  lints: ^5.0.0
  path: ^1.9.1
